modules/developer_manual/pages/app/fundamentals/info.adoc

```diff
@@ -85,55 +86,20 @@ be unique in ownCloud, meaning no other application should have this id.
 This value also represents the URL your application is available on the
 marketplace.
 
-[[category]]
-== category
-
-The category you want to publish the application in. The following
-categories are available for applications to be filed under.
-
-[width="52%",cols="50%,50%",options="header",]
-|================
-| Category Name | Value to Use
-| Automation | automation
-| Collaboration | collaboration
-| Customization | customization
-| External plugins | external-plugins
-| Games | games
-| Integration | integration
-| Multimedia | multimedia
-| Productivity | productivity
-| Security | security
-| Storage |  storage
-| Tools | tools
-|================
-
-For publishing themes this tag must be present — but empty.
-
-[source,xml]
-....
-<category></category>
-....
-
-[[description]]
-== description
-
-max. 4000 characters; This provides all the necessary, detailed
-information about the application. Don’t get lost in technical details,
-focus on the benefits which the application offers. You can use
-https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet[markdown]
-to format the description.
-
 [[name]]
 == name
 
 *Required*. This is the human-readable name (or title) of the
 application that will be displayed in the application overview page.
 
-[[description-1]]
+[[description]]
 == description
 
-*Required*. This contains the description of the application which will
-be shown in the application overview page.
+*Required*. This provides all the necessary, detailed information about the application, and is shown in the application overview page. 
+Don't get lost in technical details, focus on the benefits which the application offers. 
```

Comment:
Proposal:
For the description, don't get lost...
(eases reading, at least for me)

Reply:
I made a small amendment. However, otherwise, it reads well as it is, to me.